lib/bootstrap_form/form_group_builder.rb

```diff
@@ -77,11 +78,15 @@ def form_group_label_class(options)
       classes
     end
 
-    def form_group_required(options)
-      return unless options.key?(:skip_required)
-
-      warn "`:skip_required` is deprecated, use `:required: false` instead"
-      options[:skip_required] ? false : :default
+    def form_group_required(options, method)
+      if options[:skip_required]
+        warn "`:skip_required` is deprecated, use `:required: false` instead"
+        false
+      elsif options.key?(:required)
+        options[:required]
+      else
+        required_attribute?(object, method)
+      end
```

Comment:
I like what you did here. I wonder how it interacts with https://github.com/bootstrap-ruby/bootstrap_form/blob/fe66580854058e243251ab6eabdbc42386ff68a5/lib/bootstrap_form/components/labels.rb#L28-L33. Is some of the code there now redundant?

Reply:
I think you are right i changed it to only check for the options[:required] only one spec failed and that spec should not had have the required class anyway i think.